platform-controller/src/main/java/org/hobbit/controller/docker/ContainerManagerImpl.java

```diff
@@ -373,9 +374,13 @@ private String createContainer(String imageName, String containerType, String pa
         // benchmark (in case of the benchmark controller) and the container has
         // type "system"
         Integer numberOfSwarmNodes = Integer.MAX_VALUE;
+        Integer numberOfSystemSwarmNodes = Integer.MAX_VALUE;
+        Integer numberOfBenchmarkSwarmNodes = Integer.MAX_VALUE;
```

Comment:
I would suggest a `-1` as default value. Otherwise it is hard to figure out whether the value is set or not.

Apart from that, the values should be `int` as well :wink: